site/hxapi/fasttrack/bkg.md

```diff
@@ -41,7 +41,6 @@ NB: All parameter names are case sensitive.
 | ArrivalTime     | Time          | HHMM              | Y          | Time customer wishes to enter the fasttrack.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | Adults          | Integer       | [0-4] 1 char      | Y          | Number of adults requiring entry to the fasttrack.                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
 | Children        | Integer       | [0-3] 1 char      | N          | Number of children requiring entry to the fasttrack.                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
-| Infants         | Integer       | 1 char            | Y          | Number of infants requiring entry to the fasttrack.                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
Should it actually be removed rather than just set to not mandatory?

Reply:
I don't believe we do anything with this information when they send it, that's why I removed it.